### DISCUSSION TITLE
STASH code for skin temperature

### DISCUSSION THREAD

**Discussion by alejandrobodas** [MetOffice/simulation-systems#97] [UM]
Is there a diagnostic for skin temperature? I searched with the 'stash' command line tool and couldn't find one. Thanks!

**Comment by richardgilham** [comment]
Surface temperature do the job for you? If not, Section 3 might do you something

![image](https://github.com/MetOffice/simulation-systems/assets/52403201/eae55f54-fefd-43da-a6b6-64866875d70f)

The STASH browser might help your search (ping me for a Met Office only link; non-Met Office people, it's easy enough to set up yourself)

**Reply by Adrian-Lock** [reply]
Or stash 3316, surface temp on tiles, if you want more detail over land (rather than just grid-box average).

**Reply by alejandrobodas** [reply]
Thanks Adrian and Richard. I want the grid-box average. My doubt is whether stash 24 is the temperature of the lower layer of the atmosphere or the temperature of the surface (land & ocean). I'm interested in the latter.

**Reply by Adrian-Lock** [reply]
Stash 24 is as it says on the tin, so surface temperature, ie the temperature of the surface itself.  We ought to start a campaign to stress that temperature of the air near the surface (aka screen level or 1.5m or 2m) should never be called the surface temperature!

**Reply by alejandrobodas** [reply]
Perfect, thanks!
